hardware/MQTTAutoDiscover.cpp

```diff
@@ -50,6 +50,19 @@ MQTTAutoDiscover::MQTTAutoDiscover(const int ID, const std::string& Name, const
 	}
 }
 
+bool MQTTAutoDiscover::IsWildcardMatch(std::string st_topic, std::string m_topic)
```

Comment:
could you declare the string as const like

(const std::string &st_topic, const std::string &m_topic)

Reply:
Sure thing. I have updated that (and also the new argument to handle_auto_discovery_sensor_message)
Does the 'pull' automatically update? I think I see it does but just checking that I don't need to do something else to make the update visible.
I will get some sample message logged in a min. (Just have a work meeting...)